docs/integrations/sources/surveymonkey.md

```diff
@@ -1,48 +1,64 @@
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+Learn how to set up the SurveyMonkey source connector. A source connector extracts data from SurveyMonkey and transfers the data to a destination. Learn more about connectors [here](/using-airbyte/core-concepts/).
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for SurveyMonkey is available in the US only. We are testing how to enable OAuth in the EU. [Reach out to us](mailto:product@airbyte.io) if you run into any issues with authentication.
 
 :::
 
 <!-- env:oss -->
-## Prerequisites
+## Know before you begin
 
-**For Airbyte Open Source:**
-
-* Access Token
+You need a SurveyMonkey access token if you're using Airbyte Open Source to set up the connector. Use this token to authenticate your SurveyMonkey account with Airbyte Open Source.
 <!-- /env:oss -->
 
-## Setup guide
+## Set up the SurveyMonkey connector in Airbyte
+
 ### Step 1: Set up SurveyMonkey
-Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started). Register your application [here](https://developer.surveymonkey.com/apps/) Then go to Settings and copy your access token
+
+1. Create or log in to your [SurveyMonkey account](https://developer.surveymonkey.com/apps/).
+2. In My Apps, click **Add a New App**. The App Creation screen displays.
+3. Enter an **App Nickname** and **App Creator** email address.
+4. Select a public or private app type depending on your use case.
+5. Click **Create App**. The app overview page displays.
+6. Click **Settings**.
+7. In the Credentials section, copy your access token. You will use this token later in the guide.
+
+**Step result:** You registered a new SurveyMonkey app and access token.
+For more information about SurveyMonkey apps, read their [docs](https://developer.surveymonkey.com/api/v3/#SurveyMonkey-Api).
 
 ### Step 2: Set up the source connector in Airbyte
 
 <!-- env:cloud -->
 **For Airbyte Cloud:**
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
-3. On the source setup page, select **SurveyMonkey** from the Source type dropdown and enter a name for this connector.
-4. lick `Authenticate your account`.
-5. Log in and Authorize to the SurveyMonkey account
-6. Choose required Start date
-7. click `Set up source`.
+1. Log into your [Airbyte Cloud account](https://cloud.airbyte.com/workspaces).
```

Comment:
1. Fix many spelling and grammar mistakes using Grammarly.

2. Changed "left navigation bar" to navigation menu." In tech writing, we avoid using directional words like left, right, above, below for accessibility issues (users with vision impairment).

3. Created an Airbyte account and tested the steps. Very out-of-date. Because I need to upgrade my account, I updated the steps to the best of my ability.

4. I would love to add another sentence to step 10 to explain what a start date is.